src/Illuminate/Database/Connectors/MySqlConnector.php

```diff
@@ -23,10 +23,6 @@ public function connect(array $config)
         // connection's behavior, and some might be specified by the developers.
         $connection = $this->createConnection($dsn, $config, $options);
 
-        if (! empty($config['database'])) {
-            $connection->exec("use `{$config['database']}`;");
-        }
-
```

Comment:
In the initial implementation I dug out, there is a comment which has been removed over time (did not check why), but I guess it makes sense to check the mentioned case before fiddling around here => https://github.com/laravel/framework/commit/c203b0218e796e6b09344f6d019e1c0b7897ba10#diff-907ccb2ad354af6868ebb5618f2da86bR22-R23